sklearn/metrics/cluster/tests/test_unsupervised.py

```diff
@@ -15,31 +15,29 @@
 def test_silhouette():
     # Tests the Silhouette Coefficient.
     dataset = datasets.load_iris()
-    X = dataset.data
+    X_dense = dataset.data
+    X_sparse = csr_matrix(X_dense)
```

Comment:
I'd rather something a bit weirder, like LIL or DOK, just to be sure it's not relying on data matrix properties, etc.